CHANGELOG.rst

```diff
@@ -3,6 +3,15 @@ Changelog
 
 This document describes changes between each past release.
 
+
+8.4.0 (unreleased)
+------------------
+
+**New feature**
+
+- Add ``contains`` and ``contains_any`` filtering (Fixes #343).
```

Comment:
nitpick: usually when we add new features to the HTTP API we highlight it as such (see older entries) :)